py_mind_memo/view.py

```
@@ -638,22 +668,56 @@ def on_insert_icon(self, event):
         filemenu.add_command(label="Save As (Ctrl+Shift+S)", command=self.persistence.on_save_as)
         filemenu.add_separator()
+        filemenu.add_command(label="操作マニュアル (Ctrl+M)", command=self.on_show_manual)
+        filemenu.add_separator()
         filemenu.add_command(label="Exit", command=self.on_exit)
         menubar.add_cascade(label="File", menu=filemenu)
         self.root.config(menu=menubar)
-        
-        # ウィンドウの閉じるボタン(×)のハンドラ
-        self.root.protocol("WM_DELETE_WINDOW", self.on_exit)
+
+    def on_show_manual(self, event=None):
```

Ruff (0.15.9)

[warning] Unused method argument: `event`

(ARG002)